stories/module-analytics-components.stories.js

```diff
@@ -48,3 +60,135 @@ generateReportBasedWidgetStories( {
 	component: DashboardAllTrafficWidget,
 	wrapWidget: false,
 } );
+
+generateReportBasedWidgetStories( {
+	moduleSlug: 'analytics',
+	datastore: STORE_NAME,
+	group: 'Analytics Module/Components/Dashboard/Bounce Rate Widget',
+	data: dashboardBounceRateWidgetData,
+	options: dashboardBounceRateWidgetArgs,
+	component: DashboardBounceRateWidget,
+} );
+
+generateReportBasedWidgetStories( {
+	moduleSlug: 'analytics',
+	datastore: STORE_NAME,
+	group: 'Analytics Module/Components/Page Dashboard/Bounce Rate Widget',
+	data: dashboardBounceRateWidgetData,
+	options: dashboardBounceRateWidgetArgs,
+	component: DashboardBounceRateWidget,
+} );
+
+generateReportBasedWidgetStories( {
+	moduleSlug: 'analytics',
+	datastore: STORE_NAME,
+	group: 'Analytics Module/Components/Dashboard/Goals Widget',
+	data: dashboardGoalsWidgetData,
+	options: dashboardGoalsWidgetArgs,
+	component: DashboardGoalsWidget,
+	variantCallbacks: {
+		Loaded: ( dispatch, data, options ) => {
+			dispatch( STORE_NAME ).receiveGetReport( data, { options } );
+			dispatch( STORE_NAME ).receiveGetGoals( goals );
+		},
+		'Data Unavailable': ( dispatch ) => {
+			dispatch( STORE_NAME ).receiveGetGoals( goals );
+		},
+	},
+} );
+
+generateReportBasedWidgetStories( {
+	moduleSlug: 'analytics',
+	datastore: STORE_NAME,
+	group: 'Analytics Module/Components/Page Dashboard/Goals Widget',
+	data: dashboardGoalsWidgetData,
+	options: dashboardGoalsWidgetArgs,
+	component: DashboardGoalsWidget,
+	variantCallbacks: {
+		Loaded: ( dispatch, data, options ) => {
+			dispatch( STORE_NAME ).receiveGetReport( data, { options } );
+			dispatch( STORE_NAME ).receiveGetGoals( goals );
+		},
+		'Data Unavailable': ( dispatch ) => {
+			dispatch( STORE_NAME ).receiveGetGoals( goals );
+		},
+	},
+} );
```

Comment:
The Goals widget is only relevant site-wide, so no need for this page dashbaord-specific story.

Reply:
Updated.